src/hooks/useCourtContracts.js

```diff
@@ -442,3 +451,57 @@ export function useTotalActiveBalancePolling(termId) {
 
   return totalActiveBalance
 }
+
+export function useTotalANTStaked() {
+  const [totalANTStaked, setTotalANTStaked] = useState(bigNum(-1))
+
+  const jurorRegistryContract = useCourtContractReadOnly(
+    CourtModuleType.JurorsRegistry,
+    jurorRegistryAbi
+  )
+  useEffect(() => {
+    if (!jurorRegistryContract) {
+      return
+    }
+    const getTotalANTStaked = async () => {
+      retryMax(() => jurorRegistryContract.totalStaked())
+        .then(totalStaked => {
+          setTotalANTStaked(totalStaked)
+        })
+        .catch(err => {
+          console.error(`Error fetching ANT staked: ${err}`)
+        })
+    }
+
+    getTotalANTStaked()
+  }, [jurorRegistryContract])
+
+  return totalANTStaked
+}
+
+export function useTotalActiveBalance() {
+  const [totalActiveBalance, setTotalActiveBalance] = useState(bigNum(-1))
+  const jurorRegistryContract = useCourtContractReadOnly(
+    CourtModuleType.JurorsRegistry,
+    jurorRegistryAbi
+  )
+
+  useEffect(() => {
+    if (!jurorRegistryContract) {
+      return
+    }
+    const getTotalActiveBalance = async () => {
+      retryMax(() => jurorRegistryContract.totalActiveBalance())
+        .then(totalActive => {
+          setTotalActiveBalance(totalActive)
+        })
+        .catch(err => {
+          console.error(`Error fetching ANT staked: ${err}`)
+        })
+    }
+
+    getTotalActiveBalance()
+  }, [jurorRegistryContract])
+
+  return totalActiveBalance
+}
```

Comment:
There are a couple of updates we should do on the changes to this file. This is a good chance for @fabriziovigevani to teach what he's done in #186 :).

One thing I will say though; it would be nice for these stats to be updated every 10-15s :).

Reply:
Yes! From https://github.com/aragon/court-dashboard/pull/186#discussion_r395175180 i think we could deprecate `useTotalBalancePolling` (since we are only using it for the account banner which passes the current term) and only use this function implemented by @rperez89. 

You can see how the interval and cancel pattern has been implemented at https://github.com/aragon/court-dashboard/blob/2238e9eeed14604282fb199c431fc49917753dbd/src/hooks/useCourtContracts.js#L516
What we could also do is pass the polling frequency as parameter to the function.